reconcile/quay_repos.py

```diff
@@ -45,7 +45,7 @@ def fetch_current_state(quay_api_store):
     state = []
 
     for org_key, org_info in quay_api_store.items():
-        if not org_info['managedRepos']:
+        if not org_info['managedRepos'] and not org_info['mirror']:
```

Comment:
should this and corresponding condition in the `fetch_desired_state()` be the same?

Reply:
:thinking:

Reply:
never mind